### DISCUSSION TITLE
Didnt know where to ask, i thought may be hear

### DISCUSSION THREAD

**Discussion by Formingus** [Devolutions/UniGetUI#1745] [Q&A]
Where it save download files, even temporarily?

**Comment by marticliment** [comment]
Hello,

The download process is not handled by WingetUI, and therefore this depends on the Package Manager used to download the packages

**Reply by Formingus** [reply]
I am testing and looks very good and promising
Where to find where packager stores files, and why Microsoft framework can not be updated?

**Reply by marticliment** [reply]
this is a bug with winget, the provider of this updates. You may want to blacklist this packages from WingetUI (right click -> ignore updates)

**Comment by Formingus** [comment]
![2024-01-26_142144](https://github.com/marticliment/WingetUI/assets/37929267/0a5e7c49-b846-4975-a582-6ffc61a36b6c)
